container/functional/keylime_agent_container-basic-attestation/Dockerfile

```diff
@@ -0,0 +1,4 @@
+
+RUN dnf install -y keylime-agent-rust util-linux-core
```

Comment:
add `&& dnf clean all` to reduce image size.
Btw, should we store Dockerfile in the `test-helpers` directory? WDYT?

Reply:
If we plan use this Docker file in similar future tests, then it could be good idea to have it there.